extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -70,16 +93,25 @@ const Track = memo( ( { track, isActive, isPlaying, isError, selectTrack, index
 					selectTrack( index );
 				} }
 				onKeyDown={ e => {
-					// Only handle the Space key.
-					if ( event.key !== ' ' ) {
-						return;
+					// spacebar
+					if ( e.keyCode === 32 ) {
+						e.preventDefault();
+						selectTrack( index );
+						e.currentTarget.focus();
+					}
+					// arrow down / arrow right
+					else if ( e.keyCode === 40 || e.keycode === 39 ) {
+						e.preventDefault();
+						// send focus to the next item (or the first if none left)
+						focusTrack( trackRefs, index + 1 );
```

Comment:
In order to let the `focusTrack` handle focusing the refs, we have to pass `trackRefs` here.

My ideal would be something structured more like:

```
const trackRefs = [];

const Track = memo( ( { track, trackRefs, isActive, isPlaying, isError, selectTrack, index } ) => {
    const trackRef = useRef( null );
    trackRefs[ index ] = trackRef;

   // onKeydown, focus the track
   focusTrack( index + 1 );
```

This doesn't work though :/